Paws4Me/Paws4Me/Display All Adoptable Pets/View Controllers/AllPetDetailViewController.swift

```diff
@@ -29,25 +33,29 @@ class AllPetDetailViewController: UIViewController {
     // MARK: - IBActions
     @IBAction private func catTappedButton(_ sender: UIButton) {
         petTypeFromButton(sender)
-        sender.changePetIconsColor()
+        sender.clearPetIconBorder(button1: dogIconButton, button2: kittenIconButton, button3: puppyIconButton)
+        sender.changePetIconsBorderColor()
         petDataViewModel.searchPetCategoryType(_: petDataViewModel.petType)
         reloadView()
     }
     @IBAction private func kittenTappedButton(_ sender: UIButton) {
         petTypeFromButton(sender)
-        sender.changePetIconsColor()
+        sender.clearPetIconBorder(button1: dogIconButton, button2: catIconButton, button3: puppyIconButton)
+        sender.changePetIconsBorderColor()
```

Comment:
So maybe something to think about in the future is how can we make this code much more reusable? Perhaps a custom component? 😄

Reply:
Will do so, thank you for the suggestion 😁